data/Wooller_2002/metadata.yml

```diff
@@ -0,0 +1,200 @@
+    with nearby, similar vegetation long unburnt (Site B). This method helped to reduce
+    the problems associated with overlapping chronosequences in which differences
+    between sites may obscure temporal patterns (Whelan 1995).
+- var_in: fire_time_from_fire_to_flowering
+  unit_in: a
+  trait_name: fire_time_from_fire_to_flowering
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: 40+
+  methods: At each site, the number of inflorescences per plant was counted in most
+    years on at least 40 plants of each species, selected at random. Thus, it was
+    possible to 312 S. J. Wooller et al. monitor the onset of flowering at Site A
+    over 19 years and at Site C over 12 and 3 years after fire and to compare them
+    with nearby, similar vegetation long unburnt (Site B). This method helped to reduce
+    the problems associated with overlapping chronosequences in which differences
+    between sites may obscure temporal patterns (Whelan 1995).
+- var_in: fire_time_from_fire_to_50%_flowering
+  unit_in: a
+  trait_name: fire_time_from_fire_to_50_percent_flowering
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: 40+
+  methods: At each site, the number of inflorescences per plant was counted in most
+    years on at least 40 plants of each species, selected at random. Thus, it was
+    possible to 312 S. J. Wooller et al. monitor the onset of flowering at Site A
+    over 19 years and at Site C over 12 and 3 years after fire and to compare them
+    with nearby, similar vegetation long unburnt (Site B). This method helped to reduce
+    the problems associated with overlapping chronosequences in which differences
+    between sites may obscure temporal patterns (Whelan 1995).
+- var_in: fire_time_from_fire_to_fruit
+  unit_in: a
+  trait_name: fire_time_from_fire_to_fruiting
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: unknown
+  methods: For each of the three species, the numbers of infructescences per plant
+    were counted at Site C1 10 years after fire, at Site A 20 years after fire and
+    at Site B more than 40 years after fire.
+- var_in: fire_time_from_fire_to_50%_fruit
+  unit_in: a
+  trait_name: fire_time_from_fire_to_50_percent_fruiting
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: unknown
+  methods: For each of the three species, the numbers of infructescences per plant
+    were counted at Site C1 10 years after fire, at Site A 20 years after fire and
+    at Site B more than 40 years after fire.
+- var_in: seedbank_location
+  unit_in: .na
+  trait_name: seedbank_location
+  entity_type: population
+  value_type: mode
+  basis_of_value: expert_score
+  replicates: .na
+  methods: literature, expert knowledge
+- var_in: recruitment_timing
+  unit_in: .na
+  trait_name: recruitment_time
```

Comment:
Where did the `recruitment_time` data come from?

Reply:
from here: "the seeds of all three Banksia's germinated during the winter rains following a fire" - which is great, except that my brain was stuck in the northern hemisphere so I coded in the wrong months. Fixing now!